test/test.js

```diff
@@ -40,15 +86,195 @@ describe("ERC721RExample", function () {
     expect(balanceAfterRefundOfOwner).to.be.equal(1);
   });
 
-  it("Freely minted NFTs cannot be refunded", async function () {
+  /**
+   * OwnerMint Test
+   */
+  it("[OwnerMint] Should able to mint", async function () {
     await erc721RExample.ownerMint(1);
-    expect(await erc721RExample.isOwnerMint(1)).to.be.equal(true);
-    await expect(erc721RExample.refund([1])).to.be.revertedWith(
+    expect(await erc721RExample.balanceOf(owner.address)).to.be.equal(1);
+    expect(await erc721RExample.ownerOf(0)).to.be.equal(owner.address);
+  });
+
+  /**
+   * PublicMint Test
+   */
```

Comment:
wondering if we need all these comments. seems like a `describe` block is what you want perhaps?

Reply:
also at the start of every test name. i'd potentially just start each test with `should x` and use describe blocks to group instead